03 Writing Algorithms/14 Datasets/04 Morningstar/01 US Fundamental Data/06 Data Point Attributes.html

```diff
@@ -34,8 +34,8 @@ <h4>MorningstarIndustryCode Enumeration</h4>
 <div data-tree="QuantConnect.Data.Fundamental.MorningstarIndustryCode"></div>
 
 
-<h4>Exchange ID Values</h4>
-<p>The exchange ID represents the exchange that lists the Equity. To access the exchange ID of an Equity, use the <b>PrimaryExchangeID</b> property.</p>
+<h4>Exchange Id Values</h4>
+<p>The exchange Id represents the exchange that lists the Equity. To access the exchange Id of an Equity, use the <b>PrimaryExchangeID</b> property.</p>
 <div class="section-example-container">
     <pre class="csharp">filteredFine = fine.Where(x =&gt; x.CompanyReference.PrimaryExchangeID == "NAS");</pre>
```

Comment:
Shame this one is `PrimaryExchangeID`; it is code generated, we could make a sugar wrapper => Id to maintain backward compatibility.